flair/models/sequence_tagger_model.py

```diff
@@ -920,12 +921,13 @@ def _fetch_model(model_name) -> str:
 
             try:
                 model_path = hf_hub_download(
-                    model_name,
-                    hf_model_name,
+                    repo_id=model_name,
+                    filename=hf_model_name,
                     revision=revision,
                     library_name="flair",
                     library_version=flair.__version__,
                     cache_dir=flair.cache_root / "models" / model_folder,
+                    use_auth_token=True,
```

Comment:
Hey there! I added this to enable support for private repos, but now I realize that this will make it fail if users have not logged in. How we handle this usually is by having a `use_auth_token` param but I don't think it will fit nicely the downstream implementation in `flair`, so I'm ok if we decide to drop this as it's not needed for the pushing

Reply:
Agree, maybe we can find a better solution for private repos in a separate PR.

Reply:
Sounds good, I removed it!